### ISSUE TITLE
I got this error while running the 01 intro file 

### ISSUE THREAD

**Issue by 1030ahmad1030** [geopandas/scipy2018-geospatial-data#27]
at the code cell:-Countries_mercator = countries.to_crs(epsg=3395)  # or .to_crs({'init': 'epsg:3395'})
I had the following error.
RuntimeError: b'no arguments in initialization list'

**Comment by jorisvandenbossche** [comment]
That means that `pyproj` is not installed correctly

**Comment by 1030ahmad1030** [comment]
yeah, I downloaded only the zip file. could that be a problem as I dive deeper in the tutorial?
